src/Magento/FunctionalTestingFramework/Console/BaseGenerateCommand.php

```diff
@@ -31,7 +31,8 @@
 class BaseGenerateCommand extends Command
 {
     const MFTF_NOTICES = "Placeholder text for MFTF notices\n";
-    const CODECEPT_RUN_COMMAND = 'codecept:run functional ';
+    const CODECEPT_RUN = 'codecept:run ';
```

Comment:
double check the usage of space and whether run commands are working

Reply:
Fixed